INSTALL.md

```diff
@@ -1,90 +1,81 @@
-# Package Locations
+# Installation
 
-## PackageCloud
+Prebuilt `deb` and `rpm` packages are available via [packagecloud.io](https://packagecloud.io/). These files are also available to download directly, via the [releases page](https://github.com/Instrumental/instrumentald/releases).
 
-Prebuilt `deb` and `rpm` packages are available via the [packagecloud.io](https://packagecloud.io/) service. These files are also available to download directly:
+## OSX/macOS
 
-* 64-bit Debian package (Ubuntu, Debian) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.deb](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.deb)
-* 32-bit Debian package (Ubuntu, Debian) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.deb](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.deb)
-* 64-bit RPM package (RHEL, Amazon AMI) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.rpm](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.rpm)
-* 32-bit RPM package (RHEL, Amazon AMI) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.rpm](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.rpm)
-* 64-bit Linux tarball (CoreOS, etc.) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86_64.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86_64.tar.gz)
-* 32-bit Linux tarball (CoreOS, etc.) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86.tar.gz)
-* 64-bit Mac OS X tarball [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_osx.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_osx.tar.gz)
+Use pkg installer, then configure `/etc/instrumentald.toml` and run:
```

Comment:
I think a header for this like _**Prebuilt `pkg` file**_ would be good, or given it's repeated below, delete?